### PR TITLE
Book documentation

### PR DESCRIPTION
**Pull request by marcasmar1**

### CODE REVIEW

File: docs/src/basic_concepts/record_fields.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Record organization
+The format uses various fields to organize and distinguish **records**
+and **record sets**[^1]:
+`name`, `uas_id` and `record_number`.
+
+### Records
+**Records** are used to store data both in **detection** as well as **tracking**.
+Each record must have a unique `record_number` inside **detection** or **tracking**,
+meaning that the same `record_number` can appear in both without causing conflict.
```
Comment (aleokdev) [inline docs/src/basic_concepts/record_fields.md]: image
Bold text is way too distracting, use it sparingly. Don't use it on detection/tracking here.

Comment (aleokdev) [inline docs/src/basic_concepts/record_fields.md]:
```suggestion
Each record must have a unique `record_number` per set, be it detection or tracking.
```

**Review by aleokdev — CHANGES_REQUESTED** [review]

File: docs/src/basic_concepts/chapter_1.md [hunk 2]
```diff
@@ -4,10 +4,9 @@ As a result, it **<ins>should not be used as a reference but
 a guide<ins>**.
 
 The main procedure should be to:
-- Log data either in detection or tracks depending on
-[the nature of the data](detec_track.md).
-- Organize and distinguish elements through [record fields](record_fields.md)
-- [Classify and identify](class_ident.md) objects
+- Log data either in detection or tracks depending on the nature of the data.
+- Organize and distinguish elements through record fields
+- Classify and identify objects
```
Comment (aleokdev) [inline docs/src/basic_concepts/chapter_1.md]: Use full stops consistently

File: docs/src/basic_concepts/record_fields.md [hunk 3]
```diff
@@ -1,28 +1,22 @@
 ## Record organization
-The format uses various fields to organize and distinguish **records**
-and **record sets**[^1]:
-`name`, `uas_id` and `record_number`.
+The format uses different structures to organize and distinguish data.
```
Comment (aleokdev) [inline docs/src/basic_concepts/record_fields.md:2]: This doesn't say much, might as well remove it

File: docs/src/basic_concepts/record_fields.md [hunk 4]
```diff
@@ -1,28 +1,22 @@
...
 
 ### Records
-**Records** are used to store data both in **detection** as well as **tracking**.
-Each record must have a unique `record_number` inside **detection** or **tracking**,
-meaning that the same `record_number` can appear in both without causing conflict.
+**Records** represent all the information related with an entity in a given time and point in space.
```
Comment (aleokdev) [inline docs/src/basic_concepts/record_fields.md]: Point in space is optional in detection sets so say "in a given time and, when applicable, point in space" or something like that

File: docs/src/basic_concepts/record_fields.md [hunk 5]
```diff
@@ -1,28 +1,22 @@
...
+Each record must have a unique `record_number`.
```
Comment (aleokdev) [inline docs/src/basic_concepts/record_fields.md]: This is already explained in the reference so this doesn't add much, might as well remove it too

File: docs/src/common_uses/intercepting.md [hunk 6]
```diff
@@ -3,20 +3,19 @@
...
 
 ### 1. UAV logging
 When a UAV is detected or tracked, the system should log the UAV position
-instances in the same **record set[^1]** under a unique `uas_id`.
+instances in the same **record set**[^1] under a unique `uas_id`.
 
 ### 2. GCS logging
-The GCS location must be logged under a different **record set[^1]**, but should
-contain the same `uas_id` as the uav records, so as to link both.
+The GCS location must be logged under a different **record set**[^1], but should
```
Comment (aleokdev) [inline docs/src/common_uses/intercepting.md]: Remove the bold applied to record set here

File: docs/src/common_uses/intercepting.md [hunk 7]
```diff
@@ -3,20 +3,19 @@
...
+contain the same `uas_id` as the UAV records, so as to link both.
 
 ### 3. UAV Home location
-In the case the C-UAS is able to extract the **home location** of the uav,
-it should be included in the uav **record set[^1]**.
+In the case of C-UAS systems that are capable of extracting the **home location** of the UAV,
+it should be included in the UAV **record set**[^1].
```
Comment (aleokdev) [inline docs/src/common_uses/intercepting.md]: ... using the `home_location` member or something like that

**Review by aleokdev — COMMENTED** [review]
Much better

File: docs/src/basic_concepts/record_fields.md (hunk 3 above)
Comment (marcasmar1) [reply docs/src/basic_concepts/record_fields.md:2]: I agree that it doesn't provide much, but won't it feel a bit blunt without some sort of brief text?

Comment (aleokdev) [reply docs/src/basic_concepts/record_fields.md:2]: Then I suggest something like `most data is given as _records_, which combine with extra information to create _record sets_`.

File: docs/src/basic_concepts/detec_track.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+### Tracking
+**Tracking** is used for targets that the C-UAS has locked on and is 
+actively tracking their positions.
+These records should describe the trajectory of a specific target. 
+
+> When the system locks on a target and starts to track its position, the system should
+log each continuous track as an element of `tracks`. 
+In the case of a target coming in and out of view, each continuous track should be logged
+separately under the same `uas_id`, as a way to link the tracks.
```
Comment (aleokdev) [inline docs/src/basic_concepts/detec_track.md:14]: Detection references tracking, so put tracking first

Comment (aleokdev) [inline docs/src/basic_concepts/detec_track.md]: Add references to record sets: Detection contains a record set, tracking contains a record set.

Comment (marcasmar1) [reply docs/src/basic_concepts/detec_track.md]: (Line 3-4)
> Data inside `tracks` and `detection` is organized in record sets
(called *tracks* and *detection sets*, respectively).

Isn't this enough?

Comment (aleokdev) [reply docs/src/basic_concepts/detec_track.md]: Ah yes, hadn't seen that.

**Review by aleokdev — APPROVED** [review]

**Pull request merged**